lib/netbox_client_ruby/api/ipam/ip_address.rb

```diff
@@ -34,7 +34,7 @@ def interface
 
         return nil unless interface_data
 
-        if interface_data.key?('virtual_machine')
+        if interface_data.key? ('virtual_machine') && !interface_data['virtual_machine'].nil?
```

Comment:
Be careful of the space between `key?` and `('virtual_machine')`

Reply:
I changed the implementation to a, in my opinion, simpler one.